Update readme with migration notice to CCI (#254)
conan-io/wishlist: README.md

# Conan Wishlist

> ## DEPRECATION NOTICE: Go to [Conan Center Index instead](https://github.com/conan-io/conan-center-index)

Reviewer: ```suggestion
## DEPRECATION NOTICE: Use  [Conan Center Index repository](https://github.com/conan-io/conan-center-index) instead.
```

> # Go to [Conan Center Index instead](https://github.com/conan-io/conan-center-index)

Reviewer 2: ```suggestion
## Conan libraries and code wishlist

# DEPRECATION NOTICE

If you want to request a new package, please go to [Conan Center Index](https://github.com/conan-io/conan-center-index) instead.
```

This repository tracked user wishes for new Conan recipes prior to the start of the [Conan Center Index](https://github.com/conan-io/conan-center-index).

> Please go there now. If a Conan package does not exist in the Index yet, please search the issues first, if someone else already made a request for the library or tool you would like to see. If such a request does not exist, feel free to create one.

Reviewer 2: ```suggestion
- If a Conan package does not exist in the Index yet, please search the issues first;
- If someone else already made a request for the library or tool you would like to see;
- If such a request does not exist, feel free to [create](https://github.com/conan-io/conan-center-index/issues/new?labels=library+request&template=package_request.md&title=%5Brequest%5D+%3CLIBRARY-NAME%3E%2F%3CLIBRARY-VERSION%3E) one.
```

Author: Actually, that semicolon makes no sense there 🤔

Reviewer 2: sorry! it's Portuguese grammar :sweat_smile: It was automatic
